adala/runtimes/_litellm.py

```diff
@@ -464,7 +466,13 @@ async def batch_to_batch(
                 )
 
                 # Catch case where the model does not return a properly formatted output
-                if type(e).__name__ == "ValidationError" and "Invalid JSON" in str(e):
+                # AttributeError is an instructor bug: https://github.com/instructor-ai/instructor/pull/1103
+                # > AttributeError: 'NoneType' object has no attribute '_raw_response'
+                if type(e).__name__ in {"ValidationError", "AttributeError"}:
+                    logger.error(
+                        f"Converting error to ConstrainedGenerationError: {str(e)}"
+                    )
+                    logger.error(f"Traceback:\n{traceback.format_exc()}")
```

Comment:
let's keep logger.error for the error string and logger.debug() for the traceback

Reply:
makes sense, done